x/programs/cmd/simulator/cmd/key.go

```diff
@@ -6,72 +6,49 @@ package cmd
 import (
 	"context"
 	"fmt"
+	"time"
 
-	"github.com/spf13/cobra"
```

Comment:
Any particular reason to switch to arg parse over cobra?

Reply:
Yeah, the `interpreter` new subcommand parses commands iteratively and it requires to have access to a function that does the parsing from a string, which cobra didn't offered (I don't think)